src/extension.ts

```diff
@@ -65,6 +66,24 @@ export function activate(context: vscode.ExtensionContext) {
             vscode.window.showInformationMessage('Stack Protocol Server is starting, please try again later!');
         }
     });
+    let debugOptions = { execArgv: ['--nolazy', '--inspect=6009'] };
```

Comment:
I am not following what are you trying to achieve here. 

If you re trying to make 'Start' command work when selected from command palette, it should be done different way.

Reply:
In Start action you should check if context is not undefined. If it is not, then you just let it go the same way as current implementation. If it is undefined, it means it was called through command palette and you have t show quick pick for servers available in Servers view. Then get context selected from quick pick and call command with selected server handler.